### DISCUSSION TITLE
Ausência de { } na estrutura for

### DISCUSSION THREAD

**Discussion by syspone** [balta-io/forum#250] [Tira Dúvidas] [answered]
@andrebaltieri , na aula do editorHTML no método DrawScreen() houve declaração do for sem as chaves. fiz um teste com as chaves e o editor fica desconfigurado, nas aulas de controle é valido nao colocar as chaves se tiver somente uma linha de codigo. Poderia me explicar melhor.

![image](https://user-images.githubusercontent.com/48097255/216498961-8b7683e6-835a-4356-83d8-c05cf499e176.png)

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @cmsoouza, como vai?
No caso, você só pode dispensar o uso de chaves no caso do escopo conter apenas uma linha.

Se seu `for` tiver mais que uma linha, precisa usar `{ }`.
Você pode se orientar melhor pela identação (tabs) que o editor já te mostrou.

**Reply by syspone** [reply]
```cs
 public static void DrawScreen()
        {
            Console.Write("+");
            for (int i = 0; i <= 30; i++)
            {
                Console.Write("+");
            }

            Console.Write("-");
            Console.Write("\n");

            for (int lines = 0; lines <= 10; lines++)
            {
                Console.Write("|");
                for (int i = 0; i <= 30; i++)
                {
                    Console.Write("-");
                }

                Console.Write("+");
                Console.Write("\n");
            }
```

Agora entendi @andrebaltieri , neste cenário ja usando as chaves. Contudo tbm entendi a  situação da indentação.

**Reply by andrebaltieri** [reply]
💜
